cli/cmd/plugin/unmanaged-cluster/tanzu/tanzu.go

```diff
@@ -42,7 +42,7 @@ const (
 	tkgCoreRepoName       = "tkg-core-repository"
 	tkgGlobalPkgNamespace = "tanzu-package-repo-global"
 	tceRepoName           = "community-repository"
-	tceRepoURL            = "projects.registry.vmware.com/tce/main:0.10.1"
+	tceRepoURL            = "projects.registry.vmware.com/tce/main:0.10.2"
```

Comment:
@jpmcb can you look into why `tceRepoURL` is defined in 2 constants?

Related to the comment here: https://github.com/vmware-tanzu/community-edition/pull/3299#discussion_r819833782

can this be centralized?